test/langtools/tools/javac/doctree/DocCommentTester.java

```diff
@@ -919,7 +919,7 @@ void check(TreePath path, Name name) throws Exception {
         String normalize(String s) {
             String s2 = s.trim().replaceFirst("\\.\\s*\\n *@", ".\n@");
             StringBuilder sb = new StringBuilder();
-            Pattern p = Pattern.compile("\\{@(code|literal)( )?");
+            Pattern p = Pattern.compile("(?i)\\{@([a-z][a-z0-9.:-]*)( )?");
```

Comment:
Why is this change necessary? I see it is intended to impact the unknown inline tag in `TagTest.java`, but couldn't the test be written without this change?

Reply:
Previously, the only tags in which `newline at` was supported were `{@code}` `{@literal}` and  `{@snippet}` ... and this tag predated `{@snippet}` which is the main reason it's not included here.

Now `newline at` is supported in all inline tags, so the scope of this check is broadened accordingly.